scripts/pluto-ci.sh

```diff
@@ -7,5 +7,5 @@ cd pluto-message-ingestion
 # The pluto lambda uses a different version of node from main app
 # so we run it separately
```

Comment:
Interesting though that this isn’t true anymore: maybe it could now be run together? Not for right now anyway